src/config/rollup.config.js

```diff
@@ -115,7 +115,8 @@ const output = [
   },
 ]
 
-const useBuiltinConfig = !hasFile('.babelrc') && !hasFile('.babelrc.js') && !hasPkgProp('babel')
```

Comment:
Could you update this to allow for the `babel.config.js` as well?

Reply:
oh sorry, I didn't catch that. I am not really a rollback user exactly. I am copying kcd-scripts to make something that feels more natural to me. Thanks for all the effort you put into the community!!!